src/GitVersionCore/BuildServers/GitHubActions.cs

```diff
@@ -0,0 +1,65 @@
+using GitVersion.Logging;
+using GitVersion.OutputVariables;
+
+namespace GitVersion.BuildServers
+{
+    public class GitHubActions: BuildServerBase
+    {
+        // https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-environment-variables#default-environment-variables
+
+        public GitHubActions(IEnvironment environment, ILog log) : base(environment, log)
+        {
+        }
+
+        public const string EnvironmentVariableName = "GITHUB_ACTION";
```

Comment:
Should this be `GITHUB_ACTIONS`? 


GITHUB_ACTIONS | Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.
-- | --

Reply:
Makes sense, are you in a position to send a PR with the fix?

Reply:
https://github.com/GitTools/GitVersion/pull/2006